model/src/main/java/com/epam/repair/model/PatronymicDTO.java

```diff
@@ -10,7 +10,7 @@
  */
 @Entity
 @Table(name = "patronymic")
-public class Patronymic {
+public class PatronymicDTO {
```

Comment:
Here can't be `dto` object because it's DB view

Reply:
Comment accepted.